src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -90,6 +90,7 @@ public class CaseData implements MappableObject {
     private final List<Element<SolicitorDetails>> defendantSolicitors;
     private final List<Element<GARespondentResponse>> respondentsResponses;
     private final YesOrNo applicationIsCloaked;
+    private final YesOrNo applicationIsUncloackedOnce;
```

Comment:
applicationIsCloaked field is been hidden in UI level. It returns null when accessing it in backend during mid callbackhandler.